src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/CheckApplicantForPrevCrcController.java

```diff
@@ -0,0 +1,45 @@
+package ca.bc.gov.open.ecrc.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.ecrc.exception.EcrcExceptionConstants;
+import ca.bc.gov.open.ecrc.exception.WebServiceStatusCodes;
+import ca.bc.gov.open.ecrc.model.RequestCheckApplicantForPrevCrc;
+import ca.bc.gov.open.ecrc.service.EcrcServices;
+
+/**
+ * Check applicant for previous crc controller
+ * 
+ * @author sivakaruna
+ *
+ */
+@RestController
+public class CheckApplicantForPrevCrcController {
+	@Autowired
+	EcrcServices ecrcServices;
+
+	Logger logger = LoggerFactory.getLogger(CheckApplicantForPrevCrcController.class);
+
+	@PostMapping(value = "/private/checkApplicantForPrevCRC", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<String> checkApplicantForPrevCrc(@RequestBody RequestCheckApplicantForPrevCrc applicantInfo) {
+		logger.info("Check applicant for previous CRC request received {}", applicantInfo.getRequestGuid());
```

Comment:
I think this should be:
```suggestion
		logger.info("Check applicant for previous CRC request received [{}]", applicantInfo.getRequestGuid());
```

Reply:
updated